src/reduxMiddlewares.js

```diff
@@ -1,13 +1,32 @@
-const actionLogger = (logger, level = 'info') => store => next => (action) => {
-    logger.report(level, action, store.getState());
-    return next(action);
+const actionLogger = (
+    logger,
+    actionFilter = f => f,
+    stateFilter = () => null,
+    level = 'info',
+) => store => next => (action) => {
+    const filteredAction = actionFilter(action);
+    const filteredStateBA = stateFilter(store.getState());
```

Comment:
filteredStateBA?
filteredStateAA?
what does it exactly mean?

Reply:
filteredStateBA :- filteredStateBeforeAction
filteredStateAA :- filteredStateAfterAction

Reply:
Can we have some other names they aren't intuitive.